src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -237,6 +265,17 @@ class FileField extends React.Component {
         ? uiSchema['ui:title']
         : schema.title;
 
+    const allowUpload =
+      !enableShortWorkflow ||
+      (enableShortWorkflow &&
+        files.some((file, index) => {
+          const errors =
+            _.get([index, '__errors'], errorSchema) ||
+            [file.errorMessage].filter(error => error);
+
+          return errors.length > 0;
+        }));
```

Comment:
Why are uploads being allowed if there are errors?

Reply:
It currently allows it, so I wrapped this 'enhancement' under the feature flag.

![image](https://user-images.githubusercontent.com/83595736/123857944-f2ab5f00-d8f0-11eb-8447-5c63f728e348.png)